cmd/tdex/depositmarket_test.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"github.com/urfave/cli/v2"
+	"reflect"
+	"testing"
+)
+
+func TestFragmentation(t *testing.T) {
+	type args struct {
+		pair AssetValuePair
+	}
+	tests := []struct {
+		name  string
+		args  args
+		want  []uint64
+		want1 []uint64
+	}{
+		{
+			name: "test1",
```

Comment:
names should be there to let one understand what a test does. Can you find a better one?

Also I think that we should increase the test vectors to cover also edge cases.
Let's do this now instead of (hopefully not) experiencing issues with running daemons and being forced later to debug -> fix -> create such kind of tests.